README.md

```diff
@@ -214,6 +214,7 @@ If you’re using a framework or package manager that doesn’t support npm, you
 
    You'll notice in our example above that we also outline a `stylesheets`, `images` and `javascript` folder in your `assets` folder. These folders are to help organize any assets that are unique to your project and separate from the design system assets.
 
+   **Note:** The file date defaults to October 26, 1985, when you download and install this package. This is intentional. NPM sets a default date to ensure reproducible builds for npm packages.
```

Comment:
For clarity, does this **only** happen everywhere (NPM, GitHub, Website) or just a specific instance?

Reply:
It looks like installing the uswds package using `npm` will show the last modified date of the current date and time.

This is related solely to the `.tgz` compressed package

Reply:
I think the placement of the note in the "Downloading the package from github" and the new wording is enough specificity to now add any additional instruction. 

> "Files in the downloadable USWDS package..."

Would love to hear your thoughts on if we need to specify further!